src/app/src/androidTest/java/com/cmput301f21t09/budgetprojectname/UserRegisterTest.java

```diff
@@ -0,0 +1,96 @@
+package com.cmput301f21t09.budgetprojectname;
+
+import org.junit.Rule;
+import android.app.Activity;
+import android.widget.EditText;
+
+import org.junit.*;
+import static org.junit.Assert.*;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+import androidx.test.rule.ActivityTestRule;
+
+import com.robotium.solo.Solo;
+import com.google.firebase.auth.FirebaseAuth;
+
+/**
+ * Test class for the bottomNavigation which currently resides in MainActivity.
```

Comment:
I think you forgot to change this javadoc. Otherwise looks great! :)

Reply:
Oops